packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -3,7 +3,7 @@ title: Migrating to v2
 weight: 11
 ---
 
-CMS Design System v2 introduces several breaking changes, and this migration guide will outline high-level steps to upgrade to the latest version.
+The CMS Design System v2 release introduces several breaking changes, and this migration guide outlines high-level steps to upgrade. See the [release notes](https://github.com/CMSgov/design-system/releases/tag/core-2.0.0) for a detailed list of changes.
```

Comment:
Release note link not available: See the [release notes](https://github.com/CMSgov/design-system/releases/tag/core-2.0.0) for a detailed list of changes.

Reply:
yes, its because we haven't released v2 yet.